### DISCUSSION TITLE
[Satisfactory] Experimental Branch

### DISCUSSION THREAD

**Discussion by AznManDown** [GameServerManagers/LinuxGSM#3688] [Q&A] [answered]
With the recent updates to the Satisfactory Dedicated server, is there an easy way to swap to the experimental branch and keep the updates on my server rolling on that?

Or does anyone know exactly where I need to put the new -beta experimental flag for SteamCMD?

Thanks!

Edit: Nevermind, did another look through the docs and found the branch parameter for SteamCMD.

**Comment by Flux001337** [comment, marked as answer]
Would you share your knowledge with us?

only set `branch="-beta experimental"` to the .cfg ???

Thanks :)

https://docs.linuxgsm.com/steamcmd/branch

**Reply by AznManDown** [reply]
Very close! branch="experimental". Don't need the -beta.

**Reply by Flux001337** [reply]
Thanks
